test/configuration/shared.config.js

```diff
@@ -0,0 +1,24 @@
+const TEST_TYPE = process.env.TEST_TYPE.toLowerCase();
+
+const { SPEC_FILE, VS_DEBUGGER } = process.env;
+
+exports.testType = {
+    ...(TEST_TYPE === 'component' ? {
+        name: 'component',
+        services: ['chromedriver'],
+        specs: VS_DEBUGGER ? [SPEC_FILE] : [
+            'test/component/*.component.spec.js'
+        ]
+    } : []),
```

Comment:
It may still work but it should probably be like this:
```suggestion
    } : {}),
```